torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -4230,7 +4249,7 @@ c10::intrusive_ptr<Work> ProcessGroupNCCL::allreduce(
     }
   }
   TORCH_CHECK(
-      !isFloat8Type(tensor.scalar_type()),
+      !isUnsupportedFloat8(tensor.scalar_type()),
```

Comment:
Not from this PR but maybe fix the obvious typos "currenlty" -> "currently" haha